tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -763,3 +794,55 @@ def test_positive_login_ad_user_basic_roles(
             ldapsession.usergroup.search('')
         ldapsession.architecture.create({'name': name})
         assert ldapsession.architecture.search(name)[0]['Name'] == name
+
+
+@tier2
+def test_positive_login_user_password_otp(test_name, ipa_data, auth_source_ipa):
+    """Login with password with time based OTP
+
+    :id: be7eb5d6-3228-4660-aa64-c56f9f3ec5e0
+
+    :setup: Assure properly functioning IPA server for authentication
+
+    :steps: Login to server with an IPA user with time_based OTP.
+
+    :expectedresults: Log in to foreman UI successfully
+
+    """
+    password_with_otp = "{0}{1}".format(
+        ipa_data['ldap_ipa_user_passwd'],
+        generate_otp(ipa_data['time_based_secret']))
+    with Session(
+            test_name,
+            ipa_data['ipa_otp_username'],
+            password_with_otp
+    ) as ldapsession:
+        with raises(NavigationTriesExceeded):
+            ldapsession.user.search('')
+        expected_user = "{} {}".format(ipa_data['ipa_otp_username'], ipa_data['ipa_otp_username'])
+        assert ldapsession.task.read_all()['current_user'] == expected_user
+
```

Comment:
login with admin user and verifying otp user is created or not will be right assert. Please add it as well. Same below.